.travis.yml

```diff
@@ -3,13 +3,7 @@ sudo: required
 services:
```

Comment:
since you moving fossa to travis anyway, shall we use GitHub actions for building as well?

Reply:
done